dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionService.java

```diff
@@ -59,22 +62,20 @@ private void sendContributionsToDrc(List<ConcurContribEntry> contributionsList,
                 continue;
             }
 
-            // TODO: Send Contribution to DRC on line below:
-            boolean updateSuccessful = true; // hook in drc call here.
-            // handle response
-            // if successful/failure track accordingly.
-            if (updateSuccessful){
-                // If successful, we need to track that we have sent this, as it will form part of the XMLFile, and
-                // needs it's status to "sent" in MAAT.
-                successfulContributions.put(String.valueOf(contribEntry.getConcorContributionId()), currentContribution);
-            }
-            else{
+            String contributionId = String.valueOf(contribEntry.getConcorContributionId());
```

Comment:
If we're storing as an int, do we need to send it out as a string? Or can we just use ints here too?